src/screens/HomeStack/HomeNewsStack/index.tsx

```diff
@@ -48,6 +51,7 @@ const HomeNewsStack = memo(() => {
         options={{
           ...popupScreenOptions,
```

Comment:
```suggestion
          ...popupScreenOptionsWithMargin,
```

Reply:
We should reduce margins here not increase